apps/website/src/components/show-and-tell/gallery/ShowAndTellGallery.tsx

```diff
@@ -82,9 +82,9 @@ export function ShowAndTellGallery({
           updateConsent({ [content.data.consent]: true });
 
         content.element = document.createElement("div");
-        content.element.className =
-          "pointer-events-none flex flex-col items-center h-full p-0 md:p-4 lg:p-8 " +
-          (lightboxParent ? "w-[80%] mr-[20%]" : "w-[calc(100%-80px)]");
+        content.element.className = `pointer-events-none flex flex-col items-center h-full p-0 md:p-4 lg:p-8 ${
+          lightboxParent ? "w-[80%] mr-[20%]" : "w-[calc(100%-80px)]"
+        }`;
```

Comment:
Can we use `classes` here? (`${}` across multiple line like this looks really messy IMO)